### PR TITLE
Feature/python

### PR DESCRIPTION
**Pull request by Eloncase**

### CODE REVIEW

**Comment by LinneyS** [comment]
@flaminestone
Review this plz

File: web/documentserver-example/python/templates/index.html [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+                        {% endfor %}
+                    </select>
+                </td>
+                <td valign="middle" width="70%">Select user name before opening the document; you can open the same document using different users in different Web browser sessions, so you can check out multi-user editing functions.</td>
+            </tr>
+            <tr>
+                <td valign="middle" width="30%">
+                    <select class="select-user" id="language">
+                        {% for key, val in languages.items %}
+                            <option value="{{ key }}">{{ val }}</option>
```
Comment (LinneyS) [inline web/documentserver-example/python/templates/index.html]: language does not apply

File: web/documentserver-example/python/static/js/jscript.js [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+        jq("#beginEmbedded").addClass("disable");
+
+        jq("#embeddedView").attr("src", url);
+    });
+
+    jq(document).on("click", "#cancelEdit, .dialog-close", function () {
+        jq('#hiddenFileName').val("");
+        jq("#embeddedView").attr("src", "");
+        jq.unblockUI();
+    });
```
Comment (LinneyS) [inline web/documentserver-example/python/static/js/jscript.js]: document.location.reload();

File: web/documentserver-example/python/src/utils/docManager.py [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+    directory = getRootFolder(req)
+
+    files = os.listdir(directory)
+
+    fileInfos = []
+    for f in files:
+        if os.path.isfile(os.path.join(directory, f)):
+            fileInfos.append({ 'type': fileUtils.getFileType(f), 'title': f, 'url': getFileUri(f, req) })
+
+    return fileInfos
```
[the remark this answers is not in the document]
Comment (Eloncase) [reply web/documentserver-example/python/src/utils/docManager.py]: f0383279284e2cea3e623d3f44922aa30a646bb3

File: web/documentserver-example/python/readme.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+## Requirements
+
+Example is written for Python 3
```
[the remark this answers is not in the document]
Comment (Eloncase) [reply web/documentserver-example/python/readme.md]: 8fa77cb9ba2f2cdf37da1c3dac6bb0352e5d550e

File: web/documentserver-example/python/static/js/jscript.js (hunk 2 above)
Comment (Eloncase) [reply web/documentserver-example/python/static/js/jscript.js]: e55e27cee8290c8ee84a847818038225c75dc776

**Pull request merged**